README.md

```diff
@@ -1,5 +1,3 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Portforio
 
 > The second project in microverse.
```

Comment:
I know you can do a better effort to make an impressive README file. You should use the [Template provided by Microverse](https://github.com/microverseinc/readme-template) for this and all of your next projects in Microverse. Feel free to check [This Example](https://github.com/DanteAlonsoHT/awesome_books_js) about what we expect to see in your README files.

- [ ] Make an impressive README file.